app/views/steps/evidence/upload/edit.en.html.erb

```diff
@@ -7,32 +7,61 @@
 
     <p class="govuk-body">Use this page to provide:</p>
 
-    <ul class="govuk-list govuk-list--bullet govuk-!-padding-bottom-5">
+    <ul class="govuk-list govuk-list--bullet govuk-!-padding-bottom-3">
       <li>evidence your client gets a passporting benefit</li>
       <li>evidence of your client's income and outgoings</li>
       <li>evidence of your client's capital</li>
       <li>any other supporting evidence, like a freezing order or charge sheet</li>
     </ul>
 
+    <h2 class="govuk-heading-m">
+      <%= t('.upload_files_heading') %>
+      <span class="govuk-caption-m govuk-!-padding-top-2"><%= t('.upload_files_info') %></span>
+    </h2>
+
     <%= form_for @form_object, html: { id: 'dz-evidence-upload-form', class: 'dropzone' },
-                 url: crime_application_documents_path,
+                 url: bundle_documents_path(@form_object.current_document_bundle),
                  method: :post, multipart: true do |f| %>
 
+      <%= f.button t('.choose_files_button'),
```

Comment:
When js is switched off, we've ended up with double the buttons, I think this whole button section needs deleting.
![image](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/5f97d273-d4eb-4bcb-898a-2c87e3140074)

Reply:
Playing with it and switching js on and off, which 'Choose files' button shows needs to be conditional on if the fallback is used

Reply:
I was looking into this and it's easier said than done I fear. I think we will need to give the js button a hidden class that is removed by js (therefore _not_ removed if js isn't on)

Reply:
I've amended this